sunpy/instr/tests/test_lyra.py

```diff
@@ -81,34 +82,46 @@ def test_split_series_using_lytaf():
     assert split_no_lytaf[0]["subdata"].all() == dummy_data.all()
 
 
+# Create sample LYRALightCurve
+lyralc = lightcurve.LYRALightCurve.create("2014-01-01")
```

Comment:
This should be done inside a test marked as online, because it will download data.